test/gui/tst_deletFilesFolders/test.feature

```diff
@@ -42,7 +42,7 @@ Feature: deleting files and folders
         When the user deletes the file "textfile1.txt"
         And the user deletes the folder "test-folder1" 
         And the user waits for the files to sync
-        Then as "Alice" file "textfile1.txt" should not exist on the server
+        Then as "Alice" file "textfile1.txt" should not exist in the server
```

Comment:
This step is used in many other places as well. Please, change them all as well.